Documentation/template/_navbar.liquid

```diff
@@ -0,0 +1,21 @@
+{% comment %}Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.{% endcomment %}
+<nav id="autocollapse" class="navbar navbar-inverse ng-scope" role="navigation">
+  <div class="container">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#navbar">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
```

Comment:
don't think it is neccessory to use liquid syntax, as to keep the most similarity to mustache template, 

---
In reply to: [108393878](https://github.com/dotnet/docfx/pull/1500#discussion_r108393878) [](ancestors = 108393878)